Update Magento_BundleGraphQl ReadMe (#31307)
magento/magento2: app/code/Magento/BundleGraphQl/README.md

# BundleGraphQl

> **Magento_BundleGraphQl** module extends **Magento_GraphQl** and **Magento_Bundle** modules by provides type and resolver information for the GraphQl module

Reviewer: ```suggestion
**Magento_BundleGraphQl** module extends **Magento_GraphQl** and **Magento_Bundle** modules. The module provides type and resolver information for GraphQL API
```

to generate bundle product information.

## Installation details

> Before disabling or uninstalling this module, note that the following modules depends on this module:

Reviewer: ```suggestion
Before disabling or uninstalling this module, note that the following modules depend on this module:
```

- Magento_GraphQl
- Magento_Bundle
- Magento_QuoteGraphQl
- Magento_CatalogGraphQl
- Magento_Store

> For information about module installation in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.html).

Reviewer: ```suggestion
For information about enabling or disabling a module in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.html).
```

Author: Hi, @rogyar !
Thank you for review..
Changes was added.
